### PR TITLE
Update for JupyterLab 0.22

### PR DESCRIPTION
**Pull request by blink1073**
cc @athornton @danielballan @ellisonbg

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -57,15 +57,15 @@ function activateHubExtension(app: JupyterLab, palette: ICommandPalette, mainMen
 
   commands.addCommand(CommandIDs.controlPanel, {
     label: 'Control Panel',
-	caption: 'Open a the Hub control panel a new browser tab.',
+    caption: 'Open a the Hub control panel a new browser tab.',
```
Comment (athornton) [inline src/index.ts]: Probably "in a new browser tab."

File: src/index.ts [hunk 2]
```diff
@@ -57,15 +57,15 @@ function activateHubExtension(app: JupyterLab, palette: ICommandPalette, mainMen
 
   commands.addCommand(CommandIDs.controlPanel, {
     label: 'Control Panel',
-    caption: 'Open a the Hub control panel a new browser tab.',
+    caption: 'Open a the Hub control panel in a new browser tab',
```
Comment (athornton) [inline src/index.ts]: Drop that first "a" too.

**Comment by ellisonbg** [comment]
Looks good, merging!

**Comment by ellisonbg** [comment]
I will release tomorrow @athornton

**Comment by danielballan** [comment]
Thanks @blink1073.

**Pull request merged**